Tests/Unit/Helpers.swift

```diff
@@ -329,18 +329,31 @@ class MockMessageInterface: MiniAppMessageDelegate {
     var mockProfilePhoto: String? = ""
     var mockContactList: [MAContact]? = [MAContact(id: "contact_id")]
     var messageContentAllowed: Bool = false
+    var mockPointsInterface: Bool = false
```

Comment:
Just curious, is there a reason why it's necessary to add a variable for each mock instead of having a single `enableMockResponse` | `mockAllowed` (I feel it's a bit redudant)? for cases like ->
```
        if messageContentAllowed {
            completionHandler(.success("SUCCESS"))
        } else {
            completionHandler(.failure(.invalidContactId))
        }
```

Reply:
that's a good comment, I think you're right @laubengaier

Reply:
I wanted to do the same @laubengaier . But at times, I see the test cases were running in the parallel threads and causing other test cases to fail. But I have been planning to refactor them anyway, I will do it in another PR.